app/Enum/Report/ReportableSubject.php

```diff
@@ -0,0 +1,10 @@
+<?php declare(strict_types=1);
+
+namespace App\Enum\Report;
+
+enum ReportableSubject: string
+{
+    case EVENT  = 'Event';
```

Comment:
Maybe rather the class itself than a string?

Reply:
I've first done that, but changed it to the simplified string because I wanted to use this Enum as Validator for the API Endpoint. Otherwise other Apps using this endpoint need to send `App\Models\User` as a argument with the request.

Is this okay for you?

Reply:
Makes sense!